src/grisp_connect_client.erl

```diff
@@ -277,6 +291,17 @@ as_bin(Binary) when is_binary(Binary) -> Binary;
 as_bin(List) when is_list(List) -> list_to_binary(List);
 as_bin(Atom) when is_atom(Atom) -> atom_to_binary(Atom).
 
+%% @doc Calculate the next reconnection delay with exponential backoff.
+%% NextDelay = min(MaxDelay, LastDelay * Multiplier + Extra)
+%% Extra is a random number (-(V div 2) - (V rem 2)) and (V div 2),
+%% where V is the jitter value.
+next_connect_delay(undefined) ->
+    ?EXPBACKOFF_BASE_DELAY;
+next_connect_delay(LastDelay) ->
+    Jitter = ?EXPBACKOFF_JITTER,
+    Extra = rand:uniform(Jitter + 1) - (Jitter div 2) - (Jitter rem 2) - 1,
+    min(?EXPBACKOFF_MAX_DELAY, LastDelay * ?EXPBACKOFF_MULTIPLIER + Extra).
```

Comment:
By looking around the exponential back off definition I would do:
`min(MaxDelay,  Multiplier ** RetryCount + rand:uniform(Jitter)).`
That is more true to the definition. Of course I would also need to test if the backoff behaviour is random enough.

Reply:
So if we need anything more complicated it should be justified with a comment.